homeassistant/components/mqtt/client.py

```diff
@@ -309,7 +309,7 @@ class MQTT:
 
     def __init__(
         self,
-        hass: HomeAssistant,
+        hass,
```

Comment:
Removed typing partly because of mypy issues. Typing addressed in #77518.

Reply:
What mypy issue?

Reply:
homeassistant/components/mqtt/client.py:639: error: Argument 1 to "async_run_hass_job" of "HomeAssistant" has incompatible type "HassJob[[ReceiveMessage], Optional[Coroutine[Any, Any, None]]]"; expected "HassJob[[VarArg(Any), KwArg(Any)], Coroutine[Any, Any, None]]"